### DISCUSSION TITLE
Vercel alias set command returns "You don't have access to the domain" error

### DISCUSSION THREAD

**Discussion by cgero-eth** [vercel/community#953] [Help]
Hello Vercel community, I'm trying to integrate Vercel into our CI / CD.
The deployment command works fine, but when I try to set the alias for this deployment I get back the following error:
```
You don't have access to the domain [my-subdomain] under [my-team]
```
This is the command I'm running:
```
vercel -t [my-token] --scope [my-team] alias [deployment-url] [my-subdomain]
```

The interesting part is that setting the alias through the REST API works fine:
```
curl --location --request POST 'https://api.vercel.com/v2/deployments/[my-deployment-id]/aliases?teamId=[my-team]' \
--header 'Authorization: Bearer [my-token]' \
--header 'Content-Type: application/json' \
--data-raw '{
  "alias": "[my-subdomain]"
}'
```

**Note**:
- Vercel CLI 28.4.9
- The subdomain is already setup under my project

**UPDATE**:
This is the actual endpoint that is failing (running the command in debug mode):
```
#7 → GET https://api.vercel.com/v4/domains/[my-subdomain]?teamId=[my-team]
#7 ← 403 Forbidden: sfo1::25ftr-1665652249137-d27996578463
```

**Comment by swarnava** [comment]
Hi, What is the preview domain and alias that you are trying to set ?

**Reply by cgero-eth** [reply]
The `deployment-url` is the auto-generated url for a deployment (e.g. https://project-cnap35r3c-team-1.vercel.app) and the `my-subdomain` value is a subdomain like `vercel.example.com`:
```
vercel -t *** --scope team-1 alias https://project-cnap35r3c-team-1.vercel.app vercel.example.com
```
